Added travis

srinathnarayanan opened the pull request.

Commit:
added travis file

Commit:
updated readme

to include travis build details

msftclas commented:
@srinathnarayanan,
Thanks for your contribution as a Microsoft full-time employee or intern. You do not need to sign a CLA.
_Thanks,
Microsoft Pull Request Bot_

Commit:
moved travis file

updated readme

Commit:
moved travis file

Commit:
added dependecy on azure-storage-common

- Added dependency on storage's common package
- Edited/added Table specific code for SAS
- Added constants
